package.json

```diff
@@ -10,7 +10,10 @@
     "build-generator": "lerna run --scope @cuba-platform/front-generator build",
     "gen-client-scr": "npm run build-generator && lerna run --scope @cuba-platform/front-generator gen-client-scr",
     "update-client-libs": "node scripts/update-client-libs.js",
-    "bootstrap-client-app": "npm run build-client-libs && npm run pack-client-libs && npm run gen-client-scr && npm run update-client-libs"
+    "bootstrap-react-app": "npm run build-client-libs && npm run pack-client-libs && npm run gen-client-scr && npm run update-client-libs",
+    "start-cuba-app": "cd ./sample-car-rent && ./gradlew setupTomcat createDb deploy start",
```

Comment:
I suggest rename start-cuba-app -> bootstrap-cuba-app because we create database here.